tracing/setup.go

```diff
@@ -102,5 +123,11 @@ func SetupDynamicPublishingWithInitialValue(logger *zap.SugaredLogger, configMap
 		},
 		tracerUpdater)
 	configStore.WatchConfigs(configMapWatcher)
-	return nil
+	return oct, nil
+}
+
+// Deprecated: Use SetupPublishingWithDynamicConfigAndInitialValue.
+func SetupDynamicPublishingWithInitialValue(logger *zap.SugaredLogger, configMapWatcher configmap.Watcher, serviceName string, configm *corev1.ConfigMap) error {
+	_, err := SetupPublishingWithDynamicConfigAndInitialValue(logger, configMapWatcher, serviceName, configm)
+	return err
```

Comment:
We probably don't need this variation of the method that just returns an `error` since it was just added yesterday.